server.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+const port = process.env.PUBLIC_PORT;
```

Comment:
It's recommended to have a default port in case the environment variable is not set. [medium]